test/spec/services/apiServiceSpec.js

```diff
@@ -432,13 +432,20 @@ describe("APIService", function() {
       // These kinds either no longer exist or have been moved (for example, now
       // are listed under a different group).  They still exist, but should be
       // ignored in favor of the preferred alias.
-      it('should NOT return kind:HorizontalPodAutoscaler with group: extensions', function() {
+      it('should NOT return kind: HorizontalPodAutoscaler with group: extensions', function() {
         var allKinds = APIService.availableKinds(true);
         var toExclude = { group: 'extensions', kind: 'HorizontalPodAutoscaler' };
 
         expect(_.find(allKinds, toExclude)).toEqual(undefined);
       });
 
+      it('should NOT return kind: DaemonSet with group: apps', function() {
```

Comment:
I think the test needs updating. The apps group is the one it should return.

Reply:
Yup, tinkering with that & wondering why local tests are weird :/